rpcs3/rpcs3qt/game_list_frame.h

```diff
@@ -16,6 +16,7 @@
 #include <QSet>
 
 #include <memory>
+#include <optional>
```

Comment:
Whats this for

Reply:
Sorry, forgot to clean up after attempting another implementation which would've used an empty optional instead of invalid color.